librz/core/cmd_descs/cmd_meta.yaml

```diff
@@ -0,0 +1,407 @@
+        summary: Add unique comment
+        args:
+          - name: text
+            type: RZ_CMD_ARG_TYPE_STRING
+  - name: CS
+    summary: Manage metainformation spaces
+    subcommands:
+      - name: CS
+        cname: meta_space
+        summary: Create new metaspace / List all metaspaces
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_RIZIN
+          - RZ_OUTPUT_MODE_JSON
+        args:
+          - name: name
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: CS-
+        cname: meta_space_remove
+        summary: Remove metaspace
+        args:
+          - name: name
+            type: RZ_CMD_ARG_TYPE_STRING
+      - name: CS-*
+        cname: meta_space_remove_all
+        summary: Remove all metaspaces
+        args: []
+      - name: CSr
+        cname: meta_space_rename
+        summary: Rename the metaspace
+        args:
+          - name: oldname
+            type: RZ_CMD_ARG_TYPE_STRING
+          - name: newname
+            type: RZ_CMD_ARG_TYPE_STRING
+  - name: Cf
+    summary: Manage the format string metainformation
+    subcommands:
+      - name: Cf
+        cname: meta_format
+        summary: Set the format string to the current address / List all format marks
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_RIZIN
+          - RZ_OUTPUT_MODE_JSON
+        args:
+          - name: size
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: Cf-
+        cname: meta_format_remove
+        summary: Remove format string from the current address
+        args: []
+      - name: Cf-*
+        cname: meta_format_remove_all
+        summary: Remove all format string marks
+        args: []
+  - name: Cd
+    summary: Manage the raw data metainformation
+    subcommands:
+      - name: Cd
+        cname: meta_data
+        summary: Set the "data" mark to the current address / List all "data" marks
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_RIZIN
+          - RZ_OUTPUT_MODE_JSON
+        args:
+          - name: size
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+          - name: repeat
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+      - name: Cd.
+        cname: meta_data_at
+        summary: Show the data mark at the current offset
+        args: []
+      - name: Cd-
+        cname: meta_data_remove
+        summary: Remove the data mark from the current address
+        args: []
+      - name: Cd-*
+        cname: meta_data_remove_all
+        summary: Remove all data marks
+        args: []
+  - name: Ch
+    summary: Manage the "hidden" mark metainformation
```

Comment:
I would say a user would like to know what is this "hidden" mark...

Reply:
It hides the data marked with that mark.